.gitignore

```diff
@@ -75,3 +75,5 @@ etna/packages/magmaR.Rcheck*
 
 venv/
 __pycache__
+
+*.marker
```

Comment:
`make` is pretty good at dependency and build management, but it relies on files in the file system to reflect dependencies, which makes sense for building files but not for building "resources" like docker images or test requirements.

To realize these in a more disciplined manner, I made make targets called `*.marker` files that back 'implicit' requirements of work.  These files should never be committed however, they represent purely local build state.